src/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
```

Comment:
Reread [Task checklist point 3](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md). Component names examples there more meaningful.

`Filter` - the better name here that `Footer`.

Be always attentive with tasks descriptions and checklists. There **no redundant** words. The same would be in test tasks. You need to read it attentively and write the code according to it. It is important.